### PR TITLE
French README.md (README.fr.md)

### PR DESCRIPTION
**Pull request by Bobbbay**
The french version of README.md. Needs a review and discussion. 

Happy mapping, Bobbbay

### CODE REVIEW

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+[![Build Status](https://travis-ci.org/osmlab/name-suggestion-index.svg?branch=master)](https://travis-ci.org/osmlab/name-suggestion-index)
+[![npm version](https://badge.fury.io/js/name-suggestion-index.svg)](https://badge.fury.io/js/name-suggestion-index)
+
+## nom-suggestion-index
```
Comment (bhousel) [inline README.fr.md]: This should probably just say `name-suggestion-index` since it's the actual name of the project.

File: README.fr.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+### Participer!
+
+* Lisez le projet [Code de conduite](CODE_OF_CONDUCT.md) et n'oubliez pas d'être gentils les uns envers les autres.
+* Voir [CONTRIBUTING.md](CONTRIBUTING.md) pour savoir comment contribuer à cet index.
+
+Nous cherchons toujours de l'aide! Si vous avez des questions ou souhaitez contacter un responsable, envoyez une requête ping à `bhousel` sur:
+* [OpenStreetMap US Slack](https://slack.openstreetmap.us/)
+(Chaînes `# poi` ou` # general`)
```
Comment (bhousel) [inline README.fr.md]:
```suggestion
(Chaînes `#poi` ou `#general`)
```

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+* [`git`](https://www.atlassian.com/git/tutorials/install-git/) pour votre plateforme
+
+
+### Installation
+
+* Clonez ce projet, par exemple:
+  `git clone git@github.com: osmlab / nom-suggestion-index.git»
+* `cd` dans le dossier du projet,
+* Exécutez `npm install` pour installer les bibliothèques
+### À propos de l'index
```
Comment (bhousel) [inline README.fr.md]: Might need to preserve the spacing here and below for markdown to continue to work.

File: README.fr.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+Cela prend beaucoup de temps et beaucoup d'espace disque. Cela peut être fait occasionnellement par les responsables du projet.
+Vous n'avez pas besoin de suivre ces étapes pour contribuer à l'index.
+
+- Installer l'outil de ligne de commande `osmium` et le package de nœuds (peut être disponible uniquement dans certains environnements)
+  - `apt-get install osmium-tool` ou` brew install osmium-tool` ou similaire
+  - `npm install --no-save osmium`
+- [Télécharger la planète](http://planet.osm.org/pbf/)
+  - `curl -L -o planet-latest.osm.pbf https: // planet.openstreetmap.org / pbf / planet-latest.osm.pbf»
+- Préfiltrez le fichier planète pour inclure uniquement les éléments nommés avec les clés que nous recherchons:
+  - `balises osmium-filtre planet-latest.osm.pbf -R nom -o nommé.osm.pbf`
```
Comment (bhousel) [inline README.fr.md]: looks like the osmium commands got translated by whatever program you used? 🤔

File: README.fr.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+### Installation
+
+* Clonez ce projet, par exemple:
+  `git clone git@github.com: osmlab / nom-suggestion-index.git»
+* `cd` dans le dossier du projet,
+* Exécutez `npm install` pour installer les bibliothèques
+### À propos de l'index
+#### Fichiers générés (ne pas modifier):
+Fichiers prédéfinis (utilisés par les éditeurs OSM):
+* `dist / name-suggestions.json` - Préréglages de suggestion de nom
```
Comment (bhousel) [inline README.fr.md]: also here and below, make sure the extra spaces don't get added around path slashes or parts of urls.
```suggestion
* `dist/name-suggestions.json` - Préréglages de suggestion de nom
```

File: README.fr.md (hunk 1 above)
Comment (bmillemathias-1a) [reply README.fr.md]: yes, projects name must not be translated.

File: README.fr.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
+[![Build Status](https://travis-ci.org/osmlab/name-suggestion-index.svg?branch=master)](https://travis-ci.org/osmlab/name-suggestion-index)
+[![npm version](https://badge.fury.io/js/name-suggestion-index.svg)](https://badge.fury.io/js/name-suggestion-index)
+
+## nom-suggestion-index 
+#### TODO: nom-suggestion-index or index de nom-suggestion?
```
Comment (bmillemathias-1a) [inline README.fr.md]: same comment for the translation of `name-suggestion-index` here.

File: README.fr.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+Noms de marque communs canoniques pour OpenStreetMap.
+
+
+### Qu'est-ce que c'est?
+
+Le but de ce projet est de maintenir un [canonique](https://fr.wikipedia.org/wiki/Canonique_(informatique))
+liste des noms couramment utilisés pour suggérer l'orthographe et le balisage cohérents des fonctionnalités
```
Comment (bmillemathias-1a) [inline README.fr.md]: > un canonique liste des noms couramment utilisés 

is not proper french.

> une liste canonique des  noms couramment utilisés 

is bettter

File: README.fr.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+#### Fichiers de configuration (modifiez-les):
+* `config / *`
+  * `config / filters.json` - Expressions régulières utilisées pour filtrer` names_all` dans `names_keep` /` names_discard`
+* `marques / *` - Fichiers de configuration pour chaque type d'entreprise de marque, organisés par balise OpenStreetMap
+  * `marques / équipement / *. json`
+  * `marques / loisirs / *. json`
+  * `marques / boutique / *. json`
+  * `marques / tourisme / *. json`
+  * `marques / bureau / *. json`
+
```
Comment (bmillemathias-1a) [inline README.fr.md]: these are files / paths so must not be translated and must not have spaces in-between.

File: README.fr.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+  * Toutes les nouvelles entrées de `names_keep` qui ne sont pas déjà présentes dans l'index y seront ajoutées
+  * Génère de nombreux avertissements pour suggérer des mises à jour de `marques / ** / *. Json`
+
+
+### Autres commandes
+
+* `npm run wikidata` - Récupère les données utiles de Wikidata - étiquettes, descriptions, logos, etc.
+* `npm run` - Liste les autres outils disponibles
+
+### Mise à jour de `dist / names_all.json` depuis la planète
```
Comment (bmillemathias-1a) [inline README.fr.md]: > depuis la planète

I'd say
> depuis le fichier planet

File: README.fr.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+Cela prend beaucoup de temps et beaucoup d'espace disque. Cela peut être fait occasionnellement par les responsables du projet.
+Vous n'avez pas besoin de suivre ces étapes pour contribuer à l'index.
+
+- Installer l'outil de ligne de commande `osmium` et le package de nœuds (peut être disponible uniquement dans certains environnements)
+  - `apt-get install osmium-tool` ou` brew install osmium-tool` ou similaire
+  - `npm install --no-save osmium`
+- [Télécharger la planète](http://planet.osm.org/pbf/)
```
Comment (bmillemathias-1a) [inline README.fr.md]: > Télécharger la planète

I'd prefer
> Télécharger le fichier planet

File: README.fr.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+Le but de ce projet est de maintenir un [canonique](https://fr.wikipedia.org/wiki/Canonique_(informatique))
+liste des noms couramment utilisés pour suggérer l'orthographe et le balisage cohérents des fonctionnalités
+dans OpenStreetMap.
+
+[Regardez la vidéo](https://2019.stateofthemap.us/program/sat/mapping-brands-with-the-name-suggestion-index.html) de notre conférence à State of the Map US 2019 pour en savoir plus sur ce projet!
+
+
+### Parcourir l'index
+
+Vous pouvez parcourir l'index à https://nsi.guide/ pour voir quelles marques manquent de liens Wikidata ou ont des pages Wikipedia incomplètes.
```
Comment (bmillemathias-1a) [inline README.fr.md]: > à https://nsi.guide/

I'd prefer
>à l'adresse https://nsi.guide/

File: README.fr.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+- Nous pouvons analyser les données OSM pour les fonctionnalités "incorrectes" et produire des listes pour révision et nettoyage.
+
+<img width="1017px" alt="Index de suggestion de noms utilisé dans iD" src="https://raw.githubusercontent.com/osmlab/name-suggestion-index/master/docs/img/nsi-in-iD.gif"/>
+
+* L'index de suggestion de nom est utilisé dans iD lors de l'ajout d'un nouvel élément *
+
+Actuellement utilisé dans:
+* iD (voir ci-dessus)
+* [Vespucci](http://vespucci.io/tutorials/name_suggestions/)
+* Présélections JOSM disponibles
```
Comment (bmillemathias-1a) [inline README.fr.md]: > Les présélections JOSM

File: README.fr.md (hunk 4 above)
Comment (Bobbbay) [reply README.fr.md]: Sorry, translating in the middle of the night is tiring, kinda ignored the markup, just spaced everything out and translate word-by-word. Sorry about that... 😭😭😭

Comment (Bobbbay) [reply README.fr.md]: Sorry about that, I was tired and translated roughly half, read the translation over but missed the code, etc... 😢😴

File: README.fr.md (hunk 2 above)
Comment (doublah) [reply README.fr.md]: Is it best to direct people to english speaking channels in the french translation?

Comment (Bobbbay) [reply README.fr.md]: Your right, but are there French speaking slack channels, etc. that are active enough?

Comment (bhousel) [reply README.fr.md]: I think I'd prefer to steer people towards just the few channels that we are already on, so things aren't missed.

**Comment by bhousel** [comment]
This has been open for 2 weeks, so I'm just going to merge as-is and people can submit updates if needed..

**Pull request merged**